login.css

```diff
@@ -194,7 +193,7 @@ p{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
-	font-family: 'Roboto', sans-serif;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
20/08/2021: Changing the fonts style text that will display if it fail login to be Open Sans.